component/file_cache/lru_policy.go

```diff
@@ -74,9 +74,6 @@ type lruPolicy struct {
 }
 
 const (
-	// Check for file expiry in below number of seconds
-	CacheTimeoutCheckInterval = 5
```

Comment:
by taking this out, is there a new default that's taking place? was this magic number a significant factor in the race conditions? or was it just overkill?

Reply:
This constant was just never used. So I cleaned it up.
I have no idea what it was intended for.